dev-libs/libtpms/libtpms-0.8.4.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="Library providing software emultion of a TPM"
+HOMEPAGE="https://github.com/stefanberger/libtpms"
+SRC_URI="https://github.com/stefanberger/libtpms/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc"
+
+DEPEND="dev-libs/openssl:="
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-0.8.0-Remove-WError.patch"
+)
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--disable-static \
```

Comment:
autotools.eclass received EAPI-8 support few days ago, meaning this ebuild can be bumped to EAPI-8 now and it would allow to remove this --disable-static

https://mgorny.pl/articles/the-ultimate-guide-to-eapi-8.html#new-econf-passed-options

Reply:
I see new push is still EAPI-7 using --disable-static, I assume you want to keep it that way for now? (that's fine too if you prefer)

Reply:
For now, sticking with EAPI 7. EAPI 8 is still a little too new. I don;t know how well automated tests works with it yet, and I know packages.gentoo.org is having problems with it: https://bugs.gentoo.org/802039 .